src/index.js

```diff
@@ -0,0 +1,49 @@
+// Remove the following lines, if you do not need any of Bootstrap's JavaScript features
+import Popper from "popper.js";
+window.jQuery = $;
```

Comment:
That jQuery suff <_< I thought this thing was dead and defunct at this point?

Reply:
Nah, still alive and kickin'. Need it for bootstrap although we don't really use any part of bootstrap that requires it.